libgcc/config.host

```diff
@@ -397,10 +397,10 @@ case ${host} in
   *-*-solaris2*)
   enable_execute_stack=enable-execute-stack-mprotect.c
   ;;
-i[34567]86-*-mingw* | x86_64-*-mingw*)
+i[34567]86-*-mingw* | x86_64-*-mingw* | aarch64-*-mingw*)
   enable_execute_stack=config/i386/enable-execute-stack-mingw32.c
   ;;
-i[34567]86-*-cygwin* | x86_64-*-cygwin*)
+i[34567]86-*-cygwin* | x86_64-*-cygwin* | aarch64-*-cygwin*)
```

Comment:
I think it might be replaced with *-cygwin*

Reply:
Changed.